lib/src/DOtherSide.cpp

```diff
@@ -255,8 +251,9 @@ void dos_qguiapplication_clipboard_setImage(const char* text)
 
 void dos_qguiapplication_clipboard_setImageByUrl(const char* url)
 {
-    QNetworkAccessManager *manager = new QNetworkAccessManager();
-    QObject::connect(manager, &QNetworkAccessManager::finished, [](QNetworkReply *reply) {
+    auto manager = new QNetworkAccessManager;
```

Comment:
The optimization here could be creating manager in this way:
```
static thread_local QNetworkAccessManager manager;
```
Than we don't have to create new instance for every call and take care about deleting after finished request.
As it's merged it could be part of next PR optimizing that part if needed.